_chapters/00_about.md

```diff
@@ -107,7 +107,7 @@ Like we said, the fundaments of mathematics are the fundaments of thought. Categ
 
 The way we think and talk is based on intuition that develops naturally and is a very easy way to get our point across. However, that is part of the problem: sometimes intuition makes it *too easy* for us to say something that can be interpreted in many ways, some of which are wrong. For example, when I say that two things are equal, it would seem obvious to you what I meant, although it isn't obvious at all (how are they equal?, in what context?, etc.) 
 
-It is in these situations that people often resort to *diagrams* to refine their thoughts. Diagrams (even more than formulas) are ubiquitous in science and mathematics.
+It's in these situations that people often resort to *diagrams* to refine their thoughts. Diagrams (even more than formulas) are ubiquitous in science and mathematics.
```

Comment:
Why do you think the shortened form is better?

Reply:
I thought that more concise was better but I can change that if you want

Reply:
OK, I thought I'd correct the things myself, but I will drop some comments instead, maybe you will get something out of the conversations

Reply:
what conversation? what comments? I'm kind of confused.